src/databricks/labs/ucx/azure/resources.py

```diff
@@ -94,6 +115,18 @@ class AzureRoleAssignment:
     role_name: str
 
 
+@dataclass
+class AccessConnector:
+    id: str
+    name: str
+    location: str
+    type: str = "Microsoft.Databricks/accessConnectors"
```

Comment:
```suggestion
    identity_type: str
```
type can be 'SystemAssigned' or UserAssigned. let's not confuse the consumer

Reply:
The `AccessConnector.type` is "Microsoft.Databricks/accessConnectors". I'll remove it as it is redundant and solve the identity fields following your other comment.